ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -245,8 +84,31 @@ const triggerSet: TriggerSet<Data> = {
   timelineFile: 'r1s.txt',
   initData: () => ({
     actorSetPosTracker: {},
+    storedLeaps: {
+      oneTwoPaw: {},
+      quadCross: {},
+    },
   }),
   triggers: [
+    {
+      id: 'R1S Shockwave Knockback Safe Directions',
+      type: 'MapEffect',
+      netRegex: { location: ['00', '03'], flags: '00080004', capture: true },
```

Comment:
To avoid magic numbers, could these flags be added as a property to `mapEffectTileState`, and then use the property here?